portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -61,7 +64,19 @@ public void doPost(HttpServletRequest req, HttpServletResponse res) throws IOExc
 
     datastore.put(commentEntity);
 
+    Query query = new Query("Comment");
+    PreparedQuery results = datastore.prepare(query);
+
+    ArrayList<String> comments = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      String content = (String) entity.getProperty("content");
+      String comment = new String(content);
```

Comment:
I think you could simplify this by omitting the content variable and assigning comment to entity property since it's being casted to a String.

Reply:
Resolving because fixed unneeded line